mockup/patterns/resourceregistry/pattern.resourceregistry.less

```diff
@@ -90,18 +90,13 @@
     }
 
     .lessvariables,
-    .patternoptionsuse {
+    .patternoptions {
         .form{
             padding-top: 15px;
             clear: both;
         }
     }
 
-    .pat-resourceregistry .lessvariables .form {
-        padding-top: 10px;
-        clear: both;
-    }
-
```

Comment:
This rule doesn't make any sense. We are in `.pat-resourceregistry` already so the final rule is `.pat-resourceregistry .pat-resourceregistry .lessvariables .form` so it's never used. The rule that is used is the one just above with padding-top 15px.